docs/production/server.mdx

```diff
@@ -257,11 +257,15 @@ It sets up a websocket connection with a Fishjam instance and provides a simple
             case _:
                 ...
 
-    # Connect notifier to fishjam
-    notifier_task = asyncio.create_task(notifier.connect())
+    async def run_notifier():
+      notifier_task = asyncio.create_task(fishjam_notifier.connect())
```

Comment:
In line 250 we define the `notifier`. If you want to use `fishjam_notifier`, change it in both places